Documentation/AdvancedBuildInstructions.md

```diff
@@ -27,7 +27,7 @@ This will configure your keymap to German (`de`) instead of US English. See [`Ba
 
 The `Meta/serenity.sh` script provides an abstraction over the build targets which are made available by CMake. The
 following build targets cannot be accessed through the script and have to be used directly by changing the current
-directory to `Build/i686` and then running `ninja <target>`:
+directory to `Build/x86_64` and then running `ninja <target>`:
```

Comment:
I think the README.md should be updated as well to say that it's only 64-bit.

Reply:
Done.